src/content/zel-vue/select.mdx

```diff
@@ -0,0 +1,84 @@
+---
+label: Elements
+title: Select
+---
+
+import Table from '@zlab-de/zel-react/Table'
+import TableHeader from '@zlab-de/zel-react/TableHeader'
+import TableCell from '@zlab-de/zel-react/TableCell'
+import TableBody from '@zlab-de/zel-react/TableBody'
+import TableRow from '@zlab-de/zel-react/TableRow'
+
+# Select
+
+Select is a type of input users use to select one item from a list of values. In some cases the user can select multiple attributes. The selected item is always visible when the select box is closed.
+
+---
+
+## Available Props
+
+<Table title="Select Props" style={{ marginBottom: 60 }}>
+  <TableHeader>
+    <TableCell>Prop</TableCell>
+    <TableCell>Type</TableCell>
+    <TableCell>Default</TableCell>
+    <TableCell>Required</TableCell>
+  </TableHeader>
+  <TableBody>
+    <TableRow color="white">
+      <TableCell scope="row" component="th">
+        className
+      </TableCell>
+      <TableCell>string, object</TableCell>
+      <TableCell />
+      <TableCell />
+    </TableRow>
+    <TableRow color="gray">
+      <TableCell scope="row" component="th">
+        label
+      </TableCell>
+      <TableCell>string</TableCell>
+      <TableCell />
+      <TableCell>Yes</TableCell>
+    </TableRow>
+    <TableRow color="white">
+      <TableCell scope="row" component="th">
+        items
+      </TableCell>
+      <TableCell>array</TableCell>
+      <TableCell />
+      <TableCell>Yes</TableCell>
+    </TableRow>
+    <TableRow color="gray">
+      <TableCell scope="row" component="th">
+        placeholder
+      </TableCell>
+      <TableCell>string</TableCell>
+      <TableCell />
+      <TableCell>Yes</TableCell>
+    </TableRow>
+    <TableRow color="white">
+      <TableCell scope="row" component="th">
+        onChange
+      </TableCell>
+      <TableCell>func</TableCell>
+      <TableCell />
+      <TableCell>Yes</TableCell>
+    </TableRow>
+    <TableRow color="gray">
+      <TableCell scope="row" component="th">
+        selectedItem
+      </TableCell>
+      <TableCell>object</TableCell>
+      <TableCell>*This sets a default selected value</TableCell>
+      <TableCell> and will overwrite placeholder</TableCell>
+    </TableRow>
+  </TableBody>
+</Table>
+
+## Example
+
+```
```

Comment:
Example is missing.  ... now I guess it will be the same for following components. Thats's ok . ;)